db/migrate/20150122204353_create_password_resets.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePasswordResets < ActiveRecord::Migration
+  def change
+    create_table :password_resets do |t|
+      t.integer :user_id, null: false
+      t.string :token, limit: 128, null: false
+      t.datetime :expires_at, null: false
+      t.timestamps null: false
+    end
+
+    add_index :password_resets, :user_id
```

Comment:
should we index the `expires_at` column as well? Maybe change this to a compound index on `[:user_id, :expires_at]`? Might be too much...